bellows/types/basic.py

```diff
@@ -83,6 +84,14 @@ class uint64_t(uint_t):  # noqa: N801
     _size = 8
 
 
+class enum8(uint8_t):  # noqa: N801
+    _enum = True
+
+
+class enum16(uint16_t):  # noqa: N801
+    _enum = True
```

Comment:
Why do you need the _enum attribute? Seems unused here, or in your home-assistant PR.